com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/HDRenderPipeline.VolumetricCloudsLighting.cs

```diff
@@ -94,11 +42,11 @@ unsafe void SetPreconvolvedAmbientLightProbe(ref ShaderVariablesClouds cb, Volum
                 probeSH = SphericalHarmonicMath.RescaleCoefficients(probeSH, settings.ambientLightProbeDimmer.value);
```

Comment:
This PR from @JulienIgnace-Unity https://github.com/Unity-Technologies/Graphics/pull/6117/files is currently giving access on the GPU to the ambient probe. We should rely on this instead of going through the CPU and then we don't need to call "evaluateProbe" from C#. Let's talk